playbooks/tasks/portal-role-task.yml

```
@@ -363,14 +363,28 @@
 
 # Setup logrotate for webportal logs
 - block:
-    - name: Remove all skynet-webportal logrotate configs
-      command: rm -f /etc/logrotate.d/skynet-webportal-*
+    - name: List all existing skynet-webportal logrotate configs
+      ansible.builtin.find:
+        paths: /etc/logrotate.d/
+        patterns: "skynet-webportal-*"
+      register: existing_logrotate
+
+    - name: Remove all existing skynet-webportal logrotate configs
+      file:
+        state: absent
+        path: "{{ item.path }}"
+      with_items: "{{ existing_logrotate.files }}"
```

`with_items` is deprecated.

```suggestion
      loop: "{{ existing_logrotate.files }}"
```